module-cheatsheets/intro-to-R-tidyverse-cheatsheet.md

```diff
@@ -9,74 +9,97 @@
-| `ggplot2`         | [`ggplot()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/ggplot)                   | GG Plot               | Begins a plot that is finished by adding layers.        |
-| `ggplot2`         | [`geom_boxplot()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/geom_boxplot)       | Boxplot               | Creates a boxplot when combined with ggplot()            |
+| `ggplot2`         | [`ggplot()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.3.0/topics/ggplot)                   | GG Plot               | Begins a plot that is finished by adding layers.        |
+| `ggplot2`         | [`aes()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.3.0/topics/aes)                   | Aesthetic Mappings              | Designates how variables in the data object are mapped to the visual properties of the ggplot.        |             
+| `ggplot2`         | [`geom_boxplot()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.3.0/topics/geom_boxplot)       | Boxplot               | Creates a boxplot when combined with ggplot().            |
+| `ggplot2`         | [`geom_density()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.3.0/topics/geom_density)       | Density Plot               | Creates a smoothed plot when combined with ggplot() based on the computed density estimate.            |
+| `ggplot2`         | [`geom_point()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.3.0/topics/geom_point)       | Scatterplot               | Creates a scatterplot when combined with ggplot().            |
+| `ggplot2`         | [`geom_line()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.3.0/topics/geom_path)       | Line plot               | Creates a line plot when combined with ggplot() by connecting the points in order of the x axis variable.            |
+| `ggplot2`         | [`geom_hline()`](https://www.rdocumentation.org/packages/ggplot2/versions/0.9.0/topics/geom_hline)       | Horizontal line              | Annotates a plot with horizontal lines when combined with ggplot() and one of the `geom` functions used to draw the plot, for example, `geom_point()`.           |
+| `ggplot2`         | [`theme_classic()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.3.0/topics/ggtheme)               | Classic Theme   | Displays `ggplot` without gridlines. The [`ggtheme` documentation](https://www.rdocumentation.org/packages/ggplot2/versions/3.3.0/topics/ggtheme) has descriptions on additional themes that can be used.                                                                         |
+| `ggplot2`         | [`ggtheme()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.3.0/topics/ggtheme)               | Classic Theme   | Displays `ggplot` without gridlines.                                                                         |
```

Comment:
Is this actually a function by itself? I don't think we use this one, and `theme_classic` covers it well enough. I'd just get rid of this line to minimize confusion.